### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by pifragile**
update readme to better document the local parachain setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ It is forked from the [Cumulus](https://github.com/paritytech/cumulus) repositor
 
 Build sudo-patched polkadot binary
 ```bash
+cd ..
 git clone https://github.com/paritytech/polkadot
+cd polkddot
```
Comment (clangenb) [inline README.md:17]: typo

File: README.md [hunk 2]
```diff
@@ -22,7 +24,9 @@ diener update --substrate --branch polkadot-v0.9.18
 cargo build --release
 ```
 
-Place the binary into `../../bin/polkadot-0.9.18-sudo`
+Run:
+`cp target/release/polkadot ../../bin/polkadot-0.9.18`
```
Comment (clangenb) [inline README.md:28]: `-sudo` prefix is not needed?

Comment (pifragile) [reply README.md:28]: with sudo it crashed because it was looking for the one without sudo, so i renamed it

**Review by clangenb — COMMENTED** [review]
LGTM, @brenzi do you agree with the renaming of the binary?

**Review by clangenb — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (brenzi) [reply README.md:28]: Not ok. Kusama-local needs a patch as documented above. I'd like this patch to be explicitly visible

**Review by brenzi — CHANGES_REQUESTED** [review]
See connebt

File: README.md [hunk 3]
```diff
@@ -12,7 +12,9 @@ It is forked from the [Cumulus](https://github.com/paritytech/cumulus) repositor
 
 Build sudo-patched polkadot binary
 ```bash
+cd ..
```
Comment (brenzi) [inline README.md:15]: Remove this plz, don't see what good that does

File: README.md [hunk 4]
```diff
@@ -34,12 +38,14 @@ Place the binary into `../../bin/polkadot-0.9.18-sudo`
 
 ```bash
 # We need to build it from source. The one from the yarn registry does not work with our code.
+cd ..
```
Comment (brenzi) [inline README.md:41]: Dito

File: README.md (hunk 3 above)
Comment (pifragile) [reply README.md:15]: i use it to indicate that the polkadot repo should not be cloned inside the encointer-parachain repo but in the same parent directory

File: README.md (hunk 4 above)
Comment (pifragile) [reply README.md:41]: dito. here it is actually a specific requirement that the repos follow this structure.

File: README.md (hunk 2 above)
Comment (pifragile) [reply README.md:28]: i dont understand. if i run 
`node ../polkadot-launch/dist/cli.js ./polkadot-launch/launch-rococo-local-with-launch.json`

it get
```
🧹 Resolving parachain id...
2022-04-16 08:24:56 Building chain spec    
  ✓ Read parachain id for ../target/release/encointer-collator: 1003
Relay chain binary does not exist:  /Users/pigu/Dropbox/DATA/Documents/solidbit/kunden/encointer/code/bin/polkadot-0.9.18
```
could it be that the previous author of this readme is making assumptions about the `bin` folder (for example that a `polkadot-0.9.18` binary should already exist there) that are not documented?

File: README.md (hunk 3 above)
Comment (brenzi) [reply README.md:15]: Well, then that's what you should explain IMO. Your instructions don't start inside a particular path, so the cd.. Doesn't strike me as meaningful

Comment (pifragile) [reply README.md:15]: get it. when reading a readme of a repo i usually assume that the commands should be executed from the repo root. i will make this more clear

File: README.md (hunk 2 above)
Comment (brenzi) [reply README.md:28]: yes, I make these asumptions and the patched binary must be named with -sudo suffix. Without this patch, the kusama setup won't work locally

Comment (pifragile) [reply README.md:28]: ok, i will update the readme accordingly

**Review by brenzi — APPROVED** [review]

**Pull request merged**